coss/opensource_clubs/static/js/clubs_about.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function() {
+    'use strict';
+
+    // Smooth scrolling on nav links
+    $('#nav-about a.nav-link[href*="#"]').click(function() {
```

Comment:
I believe that wagtail gives the ability to content creators to add anchored links. I am not 100% sure about this but it's worth investigating.